components/tidb_query_datatype/src/codec/data_type/chunked_vec_sized.rs

```diff
@@ -0,0 +1,272 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+use super::chunked_vec_bool::ChunkedVecBool;
+use super::{ChunkRef, ChunkedVec, Evaluable, EvaluableRet, UnsafeRefInto};
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct ChunkedVecSized<T: Sized> {
```

Comment:
The separation of the data part and the bitmap part means it is not zero-copy for both encoding and decoding giving a Chunk format data. Maybe it can be improved.